backend.native/compiler/ir/backend.native/src/org/jetbrains/kotlin/backend/konan/CAdapterGenerator.kt

```diff
@@ -603,7 +601,6 @@ internal class CAdapterGenerator(val context: Context) : DeclarationDescriptorVi
     }
 
     override fun visitScriptDescriptor(descriptor: ScriptDescriptor, ignored: Void?): Boolean {
```

Comment:
why not just `override fun visitScriptDescriptor(descriptor: ScriptDescriptor, ignored: Void?) = true` ?